[COMSUP-121] Edge Functions polyfills (#861)
edgio-docs/edgio-docs: src/guides/v7/edge-functions.md

Edge Functions enable you to execute a small piece of JavaScript code on our edge servers. This code can be used to modify requests and responses as well as make additional calls to your defined origins. The benefits of using Edge Functions include enhancing performance by reducing latency, improving user experience by personalizing content, and increasing security by managing authentication and redirection at the edge. They allow you to execute your code closer to users, reducing the need to go back to the original server and thus, provide faster services.

**Key information:**

- Edge Functions assume you are familiar with our [CDN-as-Code](/guides/performance/cdn_as_code) approach for defining rules.

> - They require your {{ PORTAL }} team to be configured to use the Edge Functions feature. [Contact support]({{ HELP_URL }}) to enable this feature.

Reviewer: ```suggestion
- Edge Functions requires activation. {{ ACCOUNT_UPGRADE }}
```

> ## Getting Started {/* getting-started */}

Reviewer: Remove this line. This H2 has no content.

Edge Functions enable you to execute a small piece of JavaScript code on our edge servers. This code can be used to modify requests and responses as well as make additional calls to your defined origins. The benefits of using Edge Functions include enhancing performance by reducing latency, improving user experience by personalizing content, and increasing security by managing authentication and redirection at the edge. They allow you to execute your code closer to users, reducing the need to go back to the original server and thus, provide faster services.

**Key information:**

> - Edge Functions assume you are familiar with our [CDN-as-Code](/guides/performance/cdn_as_code) approach for defining rules.

Reviewer: ```suggestion
- This article assumes that you are familiar with our [CDN-as-Code](/guides/performance/cdn_as_code) approach for defining rules.
```

- Edge Functions requires activation. {{ ACCOUNT_UPGRADE }}

{{ prereq.md }}

## Defining Edge Functions {/* defining-edge-functions */}

> Edge Functions are defined in your `routes.js` file using the `edge_function` property on a route. The `edge_function` property accepts a string representing the path to the edge function file.

Reviewer: ```suggestion
Define an edge function within your {{ ROUTES_FILE }} file by adding the `edge_function` property to a route. The `edge_function` property accepts a string representing the path to the edge function file.
```

> When a request is received for a route that has an Edge Function assigned to it, the Edge Function is invoked. The Edge Function is passed two parameters: `request` and `context`.

Reviewer: Consider moving the second sentence to the subsequent section.
